saml2idp/views.py

```diff
@@ -18,6 +19,13 @@
 from . import registry
 from . import xml_signing
 
+# The 'schemes' argument for the URLValidator was introduced in Django 1.6. This
+# ensure that URL validation works in 1.5 as well.
+try:
+    URL_VALIDATOR = URLValidator(schemes=('http', 'https'))
```

Comment:
Do we want to limit ourselves to http(s) here? What if I want to redirect someone back to an app with a custom scheme? I'm thinking we should definitely validate if the URL is structurally correct but we might want to accept other (all) schemes.

That said I don't think it's a blocker because it does everything we need for our use case in Cloud 😃  Your call!